File: test/fixtures/node-superagent/get_with_data.js
import request from 'superagent';

request
  .get('http://localhost:28139/synthetics/api/v3/monitors')
  .query({
    'test': '2',
    'limit': '100',
    'w': '4'
  })
  .set('X-Api-Key', '123456789')
  .then(res => {
     console.log(res.body);
  });

File: test/fixtures/r-httr2/get_with_data.r
library(httr2)

request("http://localhost:28139/synthetics/api/v3/monitors") |>
  req_url_query(
    test = "2",
    limit = "100",
    w = "4"
  ) |>
  req_headers(`X-Api-Key` = "123456789") |>
  req_perform(verbosity = 1)
